pkg/collector/corechecks/systemd/systemd.go

```diff
@@ -267,7 +318,18 @@ func (c *SystemdCheck) submitBasicUnitMetrics(sender aggregator.Sender, conn *db
 	}
 	sender.Gauge("systemd.unit.active", float64(active), "", tags)
 	sender.Gauge("systemd.unit.loaded", float64(loaded), "", tags)
-	sender.Gauge("systemd.unit.uptime", float64(computeUptime(unit.ActiveState, activeEnterTimestamp, c.stats.TimeNanoNow())), "", tags)
+
+	unitProperties, err := c.stats.GetUnitTypeProperties(conn, unit.Name, dbusTypeMap[typeUnit])
+	if err != nil {
+		log.Warnf("Error getting unit unitProperties: %s", unit.Name)
```

Comment:
You've changed a lot of log lines to have the first letter in lowercase, I guess for consistency reasons. This one should be changed too then right?

Reply:
I only changed for `fmt.Errorf()` that create an error object. I got some hints from the IDE:

![image](https://user-images.githubusercontent.com/49917914/62731788-4245c080-ba23-11e9-98ac-aacd0f077559.png)

But for logs, it's more standard to have the first letter capitalised.